packages/ping-protect/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "@forgerock/ping-protect",
+  "version": "0.0.1",
+  "private": false,
+  "type": "commonjs",
+  "module": "./index.mjs",
+  "main": "./index.js",
+  "exports": {
+    ".": {
+      "import": "./index.mjs",
+      "require": "./index.js",
```

Comment:
I think this may be a good time to have a conversation regarding cjs/mjs support. My gut tells me we probably don't need cjs. But it may be worth a discussion.

Reply:
My understanding is commonjs is specifically for nodejs as its a different algorithm for how to resolve modules. In commonjs, we use require. 

You _can_ use commonjs in the browser but only through the use of a transpiler like webpack, where it converts the requires to something that is understandable by the browser, probably just module concatenation of some sort.

So if this has no intention of being run in node, I feel that theres probably not a lot of value by including the commonjs exports as part of the package. We wouldn't want to mislead someone by thinking its a node package.

Outside of just parity with existing package configuration, I don't know of a reason to support commonjs. Thats really where my thoughts are. I'm open to hearing why though, if we have a customer somehow needing to use the commonjs i'd be curious the use case as we dont have any server code that i know of!